web-app/js/portal/cart/DownloadPanel.js

```diff
@@ -143,17 +143,37 @@ Portal.cart.DownloadPanel = Ext.extend(Ext.Panel, {
             if (params && params.isNcwmsParams && params.latitudeRangeStart != undefined) {
                 var bounds = collection.getBounds();
                 var extent = new OpenLayers.Bounds(params.longitudeRangeStart, params.latitudeRangeStart, params.longitudeRangeEnd, params.latitudeRangeEnd);
-                intersect = bounds.containsBounds(extent, true, true);
+                intersect = this._intersect(extent, bounds);
             } else if (params && params.type === 'geometrypropertytype') {
                 var bounds = collection.getBounds();
                 var extent = new OpenLayers.Bounds(params.value.bounds.left, params.value.bounds.bottom, params.value.bounds.right, params.value.bounds.top);
-                intersect = bounds.containsBounds(extent, true, true);
```

Comment:
How about using Bounds.intersectsBounds instead of containsBounds rather than writing the intersection code yourself?

Refer http://dev.openlayers.org/docs/files/OpenLayers/BaseTypes/Bounds-js.html#OpenLayers.Bounds.intersectsBounds